src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -79,225 +125,472 @@ const useStyles = makeStyles()(() => ({
+                  <Grid container>
+                    <Grid item lg={4} md={4} xs={12}>
+                      <Box
+                        display="flex"
+                        p={2}
+                        alignItems="center"
+                        className={classes.left}
+                      >
+                        <ContactAvatar
+                          src={avatar || ''}
+                          aria-label="Contact Avatar"
+                        />
+                        <Box display="flex" flexDirection="column" ml={2}>
+                          <Link
+                            data-testid="contactSelect"
+                            underline="hover"
+                            onClick={() => setContactFocus(id, 'Donations')}
+                          >
+                            <Typography variant="subtitle1">{name}</Typography>
+                          </Link>
+                          <Typography variant="subtitle2">
+                            {`Current: ${statusTitle} ${
+                              amount && amountCurrency
+                                ? currencyFormat(amount, amountCurrency, locale)
+                                : ''
+                            } ${frequencyTitle}`}
+                          </Typography>
+                        </Box>
+                      </Box>
+                    </Grid>
+                    <Grid item xs={12} md={8} lg={8} className={classes.right}>
+                      <Grid
+                        container
+                        style={{ paddingRight: theme.spacing(1) }}
+                      >
+                        <Grid item xs={12} md={6} lg={12}>
+                          <Box className={classes.boxTop}>
+                            <FormControl fullWidth size="small">
+                              <InputLabel id="status-label">
+                                {t('Status')}
+                              </InputLabel>
+                              <Select
+                                className={classes.select}
+                                size="small"
+                                placeholder="Status"
+                                labelId="status-label"
+                                label={t('Status')}
+                                inputProps={{
+                                  'data-testid': 'pledgeStatus-input',
+                                }}
+                                data-testid="statusSelect"
+                                style={{ width: '100%' }}
+                                value={statusValue}
+                                onChange={(event) =>
+                                  setFieldValue(
+                                    'statusValue',
+                                    event.target.value,
+                                  )
+                                }
+                              >
+                                <MenuItem value="">Status</MenuItem>
```

Comment:
You can remove this menu item that says "Status" and remove "Frequency as a menu item as well.
<img width="850" alt="Screenshot 2024-08-09 at 8 23 23 AM" src="https://github.com/user-attachments/assets/870cde93-1724-4a74-aa76-c2209c2de8bb">